frontend/public/index.html

```diff
@@ -26,6 +26,21 @@
 
     <title>Streamlit</title>
 
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,400;0,700;1,400;1,700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Source+Code+Pro:ital,wght@0,400;0,700;1,400;1,700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Source+Serif+Pro:ital,wght@0,400;0,700;1,400;1,700&display=swap"
+      rel="stylesheet"
+    />
```

Comment:
Happy to keep making these web fonts, but I should add that in investigating, https://github.com/adobe-fonts/source-sans uses the SIL Open Font License, which is quite permissive, so we _can_ go the approach of downloading them and placing them in (I am not a lawyer).

But I won't keep pushing it if we want to do this in the meantime.

Reply:
Oh, that's great news! I was going by the Adobe page, which says you [cannot bundle their fonts](https://helpx.adobe.com/fonts/user-guide.html/fonts/using/font-licensing.ug.html), but upon re-reading it it's clear that's basically boilerplate text that sends you to look at the original license for each font. And the license in this case is very clear in saying this _is indeed_ supported for these specific fonts.

I'll change this around.